### ISSUE TITLE
export_model.py导出模型出错

### ISSUE THREAD

**Issue by ZhijunLStudio** [PaddlePaddle/PaddleClas#1911]
PaddleClas release/2.3
windows10 python3.7 CUDA10.2/cuDNN 7.6.5 paddle2.2.2
分类模型训练完成，使用导出命令时出错

python tools/export_model.py -c ppcls/configs/GeneralRecognition/GeneralRecognition_PPLCNet_x2_5.yaml -o Global.pretrained_model="output/RecModel/best_model"

[2022/05/11 14:08:40] root INFO: 
[2022/05/11 14:08:40] root INFO: Arch : 
[2022/05/11 14:08:40] root INFO:     Backbone :
[2022/05/11 14:08:40] root INFO:         name : PPLCNet_x2_5
[2022/05/11 14:08:40] root INFO:         pretrained : True
[2022/05/11 14:08:40] root INFO:         use_ssld : True
[2022/05/11 14:08:40] root INFO:     BackboneStopLayer :
[2022/05/11 14:08:40] root INFO:         name : flatten
[2022/05/11 14:08:40] root INFO:     Head :
[2022/05/11 14:08:40] root INFO:         class_num : 7
[2022/05/11 14:08:40] root INFO:         embedding_size : 512
[2022/05/11 14:08:40] root INFO:         margin : 0.2
[2022/05/11 14:08:40] root INFO:         name : ArcMargin
[2022/05/11 14:08:40] root INFO:         scale : 30
[2022/05/11 14:08:40] root INFO:     Neck :
[2022/05/11 14:08:40] root INFO:         class_num : 512
[2022/05/11 14:08:40] root INFO:         embedding_size : 1280
[2022/05/11 14:08:40] root INFO:         name : FC
[2022/05/11 14:08:40] root INFO:     infer_add_softmax : False
[2022/05/11 14:08:40] root INFO:     infer_output_key : features
[2022/05/11 14:08:40] root INFO:     name : RecModel
[2022/05/11 14:08:40] root INFO: DataLoader :
[2022/05/11 14:08:40] root INFO:     Eval :
[2022/05/11 14:08:40] root INFO:         Gallery :
[2022/05/11 14:08:40] root INFO:             dataset :
[2022/05/11 14:08:40] root INFO:                 cls_label_path : C:/Users/zhijun.li/Desktop/PaddleClas-release-2.3/dataset/val.txt
[2022/05/11 14:08:40] root INFO:                 image_root : C:/Users/zhijun.li/Desktop/PaddleClas-release-2.3/dataset
[2022/05/11 14:08:40] root INFO:                 name : VeriWild
[2022/05/11 14:08:40] root INFO:                 transform_ops :
[2022/05/11 14:08:40] root INFO:                     DecodeImage :
[2022/05/11 14:08:40] root INFO:                         channel_first : False
[2022/05/11 14:08:40] root INFO:                         to_rgb : True
[2022/05/11 14:08:40] root INFO:                     ResizeImage :
[2022/05/11 14:08:40] root INFO:                         size : 224
[2022/05/11 14:08:40] root INFO:                     NormalizeImage :
[2022/05/11 14:08:40] root INFO:                         mean : [0.485, 0.456, 0.406]
[2022/05/11 14:08:40] root INFO:                         order :
[2022/05/11 14:08:40] root INFO:                         scale : 0.00392157
[2022/05/11 14:08:40] root INFO:                         std : [0.229, 0.224, 0.225]
[2022/05/11 14:08:40] root INFO:             loader :
[2022/05/11 14:08:40] root INFO:                 num_workers : 0
[2022/05/11 14:08:40] root INFO:                 use_shared_memory : True
[2022/05/11 14:08:40] root INFO:             sampler :
[2022/05/11 14:08:40] root INFO:                 batch_size : 2
[2022/05/11 14:08:40] root INFO:                 drop_last : False
[2022/05/11 14:08:40] root INFO:                 name : DistributedBatchSampler
[2022/05/11 14:08:40] root INFO:                 shuffle : False
[2022/05/11 14:08:40] root INFO:         Query :
[2022/05/11 14:08:40] root INFO:             dataset :
[2022/05/11 14:08:40] root INFO:                 cls_label_path : C:/Users/zhijun.li/Desktop/PaddleClas-release-2.3/dataset/val.txt
[2022/05/11 14:08:40] root INFO:                 image_root : C:/Users/zhijun.li/Desktop/PaddleClas-release-2.3/dataset
[2022/05/11 14:08:40] root INFO:                 name : VeriWild
[2022/05/11 14:08:40] root INFO:                 transform_ops :
[2022/05/11 14:08:40] root INFO:                     DecodeImage :
[2022/05/11 14:08:40] root INFO:                         channel_first : False
[2022/05/11 14:08:40] root INFO:                         to_rgb : True
[2022/05/11 14:08:40] root INFO:                     ResizeImage :
[2022/05/11 14:08:40] root INFO:                         size : 224
[2022/05/11 14:08:40] root INFO:                     NormalizeImage :
[2022/05/11 14:08:40] root INFO:                         mean : [0.485, 0.456, 0.406]
[2022/05/11 14:08:40] root INFO:                         order :
[2022/05/11 14:08:40] root INFO:                         scale : 0.00392157
[2022/05/11 14:08:40] root INFO:                         std : [0.229, 0.224, 0.225]
[2022/05/11 14:08:40] root INFO:             loader :
[2022/05/11 14:08:40] root INFO:                 num_workers : 0
[2022/05/11 14:08:40] root INFO:                 use_shared_memory : True
[2022/05/11 14:08:40] root INFO:             sampler :
[2022/05/11 14:08:40] root INFO:                 batch_size : 2
[2022/05/11 14:08:40] root INFO:                 drop_last : False
[2022/05/11 14:08:40] root INFO:                 name : DistributedBatchSampler
[2022/05/11 14:08:40] root INFO:                 shuffle : False
[2022/05/11 14:08:40] root INFO:     Train :
[2022/05/11 14:08:40] root INFO:         dataset :
[2022/05/11 14:08:40] root INFO:             cls_label_path : C:/Users/zhijun.li/Desktop/PaddleClas-release-2.3/dataset/train.txt
[2022/05/11 14:08:40] root INFO:             image_root : C:/Users/zhijun.li/Desktop/PaddleClas-release-2.3/dataset
[2022/05/11 14:08:40] root INFO:             name : ImageNetDataset
[2022/05/11 14:08:40] root INFO:             transform_ops :
[2022/05/11 14:08:40] root INFO:                 DecodeImage :
[2022/05/11 14:08:40] root INFO:                     channel_first : False
[2022/05/11 14:08:40] root INFO:                     to_rgb : True
[2022/05/11 14:08:40] root INFO:                 RandCropImage :
[2022/05/11 14:08:40] root INFO:                     size : 224
[2022/05/11 14:08:40] root INFO:                 RandFlipImage : 
[2022/05/11 14:08:40] root INFO:                     flip_code : 1
[2022/05/11 14:08:40] root INFO:                 NormalizeImage :
[2022/05/11 14:08:40] root INFO:                     mean : [0.485, 0.456, 0.406]
[2022/05/11 14:08:40] root INFO:                     order :
[2022/05/11 14:08:40] root INFO:                     scale : 1.0/255.0
[2022/05/11 14:08:40] root INFO:                     std : [0.229, 0.224, 0.225]
[2022/05/11 14:08:40] root INFO:         loader :
[2022/05/11 14:08:40] root INFO:             num_workers : 0
[2022/05/11 14:08:40] root INFO:             use_shared_memory : True
[2022/05/11 14:08:40] root INFO:         sampler :
[2022/05/11 14:08:40] root INFO:             batch_size : 2
[2022/05/11 14:08:40] root INFO:             drop_last : False
[2022/05/11 14:08:40] root INFO:             name : DistributedBatchSampler
[2022/05/11 14:08:40] root INFO:             shuffle : True
[2022/05/11 14:08:40] root INFO: Global :
[2022/05/11 14:08:40] root INFO:     checkpoints : None
[2022/05/11 14:08:40] root INFO:     device : gpu
[2022/05/11 14:08:40] root INFO:     epochs : 10
[2022/05/11 14:08:40] root INFO:     eval_during_train : True
[2022/05/11 14:08:40] root INFO:     eval_interval : 10
[2022/05/11 14:08:40] root INFO:     eval_mode : retrieval
[2022/05/11 14:08:40] root INFO:     image_shape : [3, 224, 224]
[2022/05/11 14:08:40] root INFO:     output_dir : ./output/
[2022/05/11 14:08:40] root INFO:     pretrained_model : output/RecModel/best_model
[2022/05/11 14:08:40] root INFO:     print_batch_step : 10
[2022/05/11 14:08:40] root INFO:     save_inference_dir : ./inference
[2022/05/11 14:08:40] root INFO:     save_interval : 10
[2022/05/11 14:08:40] root INFO:     to_static : False
[2022/05/11 14:08:40] root INFO:     use_dali : False
[2022/05/11 14:08:40] root INFO:     use_visualdl : True
[2022/05/11 14:08:40] root INFO: Loss :
[2022/05/11 14:08:40] root INFO:     Eval :
[2022/05/11 14:08:40] root INFO:         CELoss :
[2022/05/11 14:08:40] root INFO:             weight : 1.0
[2022/05/11 14:08:40] root INFO:     Train :
[2022/05/11 14:08:40] root INFO:         CELoss :
[2022/05/11 14:08:40] root INFO:             weight : 1.0
[2022/05/11 14:08:40] root INFO: Metric :
[2022/05/11 14:08:40] root INFO:     Eval :
[2022/05/11 14:08:40] root INFO:         Recallk :
[2022/05/11 14:08:40] root INFO:             topk : [1, 5]
[2022/05/11 14:08:40] root INFO: Optimizer :
[2022/05/11 14:08:40] root INFO:     lr :
[2022/05/11 14:08:40] root INFO:         learning_rate : 0.04
[2022/05/11 14:08:40] root INFO:         name : Cosine
[2022/05/11 14:08:40] root INFO:         warmup_epoch : 5
[2022/05/11 14:08:40] root INFO:     momentum : 0.9
[2022/05/11 14:08:40] root INFO:     name : Momentum
[2022/05/11 14:08:40] root INFO:     regularizer :
[2022/05/11 14:08:40] root INFO:         coeff : 1e-05
[2022/05/11 14:08:40] root INFO:         name : L2
[2022/05/11 14:08:40] root INFO: train with paddle 2.2.2 and device CUDAPlace(0)
W0511 14:08:40.735852  6588 device_context.cc:447] Please NOTE: device: 0, GPU Compute Capability: 7.5, Driver API Version: 11.1, Runtime API Version: 10.2
W0511 14:08:40.743853  6588 device_context.cc:465] device: 0, cuDNN Version: 7.6.
[2022/05/11 14:08:43] root INFO: unique_endpoints {''}
[2022/05/11 14:08:43] root INFO: Found C:\Users\zhijun.li/.paddleclas/weights\PPLCNet_x2_5_ssld_pretrained.pdparams
C:\Users\zhijun.li\AppData\Local\conda\conda\envs\paddle\lib\site-packages\paddle\fluid\layers\utils.py:77: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Pyt
hon 3.3,and in 3.9 it will stop working
  return (isinstance(seq, collections.Sequence) and

GeneralRecognition_PPLCNet_x2_5.yaml文件内容如下：

# global configs
Global:
  checkpoints: null
  pretrained_model: null
  output_dir: ./output/
  device: gpu
  save_interval: 10
  eval_during_train: True
  eval_interval: 10
  epochs: 10
  print_batch_step: 10
  use_visualdl: True
  # used for static mode and model export
  image_shape: [3, 224, 224]
  save_inference_dir: ./inference
  eval_mode: retrieval
  use_dali: False
  to_static: False

# model architecture
Arch:
  name: RecModel
  infer_output_key: features
  infer_add_softmax: False

  Backbone: 
    name: PPLCNet_x2_5
    pretrained: True
    use_ssld: True
  BackboneStopLayer:
    name: "flatten"
  Neck:
    name: FC
    embedding_size: 1280
    class_num: 512
  Head:
    name: ArcMargin 
    embedding_size: 512
    class_num: 7
    margin: 0.2
    scale: 30

# loss function config for traing/eval process
Loss:
  Train:
    - CELoss:
        weight: 1.0
  Eval:
    - CELoss:
        weight: 1.0

Optimizer:
  name: Momentum
  momentum: 0.9
  lr:
    name: Cosine
    learning_rate: 0.04
    warmup_epoch: 5
  regularizer:
    name: 'L2'
    coeff: 0.00001

# data loader for train and eval
DataLoader:
  Train:
    dataset:
      name: ImageNetDataset
      image_root: C:/Users/zhijun.li/Desktop/PaddleClas-release-2.3/dataset
      cls_label_path: C:/Users/zhijun.li/Desktop/PaddleClas-release-2.3/dataset/train.txt
      transform_ops:
        - DecodeImage:
            to_rgb: True
            channel_first: False
        - RandCropImage:
            size: 224
        - RandFlipImage:
            flip_code: 1
        - NormalizeImage:
            scale: 1.0/255.0
            mean: [0.485, 0.456, 0.406]
            std: [0.229, 0.224, 0.225]
            order: ''

    sampler:
      name: DistributedBatchSampler
      batch_size: 2
      drop_last: False
      shuffle: True
    loader:
      num_workers: 0
      use_shared_memory: True

  Eval:
    Query:
      dataset: 
        name: VeriWild
        image_root: C:/Users/zhijun.li/Desktop/PaddleClas-release-2.3/dataset
        cls_label_path: C:/Users/zhijun.li/Desktop/PaddleClas-release-2.3/dataset/val.txt
        transform_ops:
          - DecodeImage:
              to_rgb: True
              channel_first: False
          - ResizeImage:
              size: 224
          - NormalizeImage:
              scale: 0.00392157
              mean: [0.485, 0.456, 0.406]
              std: [0.229, 0.224, 0.225]
              order: ''
      sampler:
        name: DistributedBatchSampler
        batch_size: 2
        drop_last: False
        shuffle: False
      loader:
        num_workers: 0
        use_shared_memory: True

    Gallery:
      dataset: 
        name: VeriWild
        image_root: C:/Users/zhijun.li/Desktop/PaddleClas-release-2.3/dataset
        cls_label_path: C:/Users/zhijun.li/Desktop/PaddleClas-release-2.3/dataset/val.txt
        transform_ops:
          - DecodeImage:
              to_rgb: True
              channel_first: False
          - ResizeImage:
              size: 224
          - NormalizeImage:
              scale: 0.00392157
              mean: [0.485, 0.456, 0.406]
              std: [0.229, 0.224, 0.225]
              order: ''
      sampler:
        name: DistributedBatchSampler
        batch_size: 2
        drop_last: False
        shuffle: False
      loader:
        num_workers: 0
        use_shared_memory: True

Metric:
  Eval:
    - Recallk:
        topk: [1, 5]

**Comment by weisy11** [comment]
你好，你这个log看起来并没有报错啊
